src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ImportCompletionItem.cs

```diff
@@ -38,6 +39,11 @@ public static CompletionItem Create(string name, int arity, string containingNam
                 {
                     builder.Add(MethodKey, extensionMethodData.Value.methodSymbolKey);
                     builder.Add(ReceiverKey, extensionMethodData.Value.receiverTypeSymbolKey);
+
+                    if (extensionMethodData.Value.overloadCount > 0)
```

Comment:
would maybe have been nice to have this in another pr.

Reply:
All changes related to this is in the last commit, skipping it would make reviewer easier.